### DISCUSSION TITLE
Accessing keys

### DISCUSSION THREAD

**Discussion by arnelamo** [i18next/i18next#1743] [General]
Hi. I just started using i18next, and have previously used 'react-intl'. We're using Typescript, and I was thinking it would be nice to get suggestions when accessing nested keys, but that seems difficult as long as I have to access the keys like this:

`t('someKey.nestedKey')`

What I would like is to somehow get suggestions as soon as you start typing the key. Any ideas if/how this can be done?

**Comment by pedrodurek** [comment]
Hey @arnelamo, the implementation it's underway, I'll let you know once I finish it 😄

**Reply by zorzysty** [reply]
Hi @pedrodurek! 
Any update on this? This would be great to have it :) Cheers!
